Nos.xcodeproj/project.pbxproj

```diff
@@ -2936,7 +2936,7 @@
 				CURRENT_PROJECT_VERSION = 224;
 				DEVELOPMENT_TEAM = GZCZBKH7MY;
 				GENERATE_INFOPLIST_FILE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 16.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 17.0;
```

Comment:
Do we want to call this out in the CHANGELOG, too? Or is this a case where we were actually targeting iOS 17 already but these were set to 16.2?

Reply:
Oh I see now that it was just the performance test target. No need for a CHANGELOG update, I think.